guide/src/migration.md

```diff
@@ -1,5 +1,37 @@
 # Appendix B: Migrating from older PyO3 versions
-This guide can help you upgrade code through breaking changes from one PyO3 version to the next. For a detailed list of all changes, see [CHANGELOG.md](https://github.com/PyO3/pyo3/blob/master/CHANGELOG.md)
+
+This guide can help you upgrade code through breaking changes from one PyO3 version to the next.
+For a detailed list of all changes, see [CHANGELOG.md](https://github.com/PyO3/pyo3/blob/master/CHANGELOG.md)
+
+## from 0.9.* to 0.10
+
+### `ObjectProtocol` is removed
```

Comment:
Is it worth noting that `ObjectProtocol` was also in the prelude? So for many users this change may have been totally invisible.